lib/Conversion/AIEToConfiguration/AIEToConfiguration.cpp

```diff
@@ -437,7 +437,8 @@ xilinx::AIE::convertTransactionBinaryToMLIR(mlir::MLIRContext *ctx,
   // create aie.device
   std::vector<AIEDevice> devices{AIEDevice::npu1_1col, AIEDevice::npu1_2col,
                                  AIEDevice::npu1_3col, AIEDevice::npu1};
-  auto device = builder.create<DeviceOp>(loc, devices[columns - 1]);
+  auto device = builder.create<DeviceOp>(
+      loc, devices[columns - 1], StringAttr::get(builder.getContext(), "main"));
```

Comment:
Is the intent to use the default name here? I know the `getDefaultName()` doesn't exist, but maybe like,
```suggestion
      loc, devices[columns - 1], StringAttr::get(builder.getContext(), DeviceOp::getDefaultName()));
```
 Or have a constructor that fills in the default,
```suggestion
      loc, devices[columns - 1]);
```